pkg/frontend/mysql_cmd_executor.go

```diff
@@ -1116,6 +1116,44 @@ func (mce *MysqlCmdExecutor) handleSetVar(sv *tree.SetVar) error {
 	return nil
 }
 
+func (mce *MysqlCmdExecutor) handleShowErrors() error {
+	var err error = nil
+	ses := mce.GetSession()
+	proto := mce.GetSession().protocol
+
+	levelCol := new(MysqlColumn)
+	levelCol.SetColumnType(defines.MYSQL_TYPE_VARCHAR)
+	levelCol.SetName("Level")
+
+	CodeCol := new(MysqlColumn)
+	CodeCol.SetColumnType(defines.MYSQL_TYPE_SHORT)
+	CodeCol.SetName("Code")
+
+	MsgCol := new(MysqlColumn)
+	MsgCol.SetColumnType(defines.MYSQL_TYPE_VARCHAR)
+	MsgCol.SetName("Message")
+
+	ses.Mrs.AddColumn(levelCol)
+	ses.Mrs.AddColumn(CodeCol)
+	ses.Mrs.AddColumn(MsgCol)
+
+	for i := len(ses.moerrs) - 1; i >= 0; i-- {
+		row := make([]interface{}, 3)
+		row[0] = "Error"
+		row[1] = ses.moerrs[i].ErrorCode()
+		row[2] = ses.moerrs[i].Error()
+		ses.Mrs.AddRow(row)
```

Comment:
Is this transferring error to clinet?     If it is last step, we may want to either include, or use the moerr.MySQLCode().

Reply:
done